perf_dashboard/helpers/outlier_detection.py

```diff
@@ -0,0 +1,79 @@
+# Copyright Istio Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+def find_outliers(pattern_data):
+    perf_val = []
+    for data in pattern_data:
+        if data[3] != 'null':
+            perf_val.append((data[3], data[4], data[5]))
```

Comment:
what is in 3, 4, 5? Should we make it an object so we can refer to it by data.foo, data.bar, etc?

Reply:
yes, good catch.